Sources/Container.swift

```diff
@@ -178,16 +178,7 @@ public final class Container {
     }
 
     internal func restoreObjectGraph(_ identifier: GraphIdentifier) {
```

Comment:
True, and if it weren't for the tests we could mark this private too. Or discard the function and just set it—since this function is only called within this file at this point. I will add a comment.